### PR TITLE
One-hit Obliterator Object feature

### PR DESCRIPTION
**Pull request by Kaedenn**
Moved "Master Mode" icon to a separate line. Added "One-hit Obliterator Object" icon to another line below. Added an attribute which is populated automatically by the unserialiser.

---
This change is [Reviewable](https://reviewable.io/reviews/zeldamods/objmap/74)

### CODE REVIEW

File: src/components/ObjectInfo.vue [hunk 1]
```diff
@@ -9,7 +9,12 @@
       <i class="fas fa-hashtag fa-fw"></i> ID
       <span v-if="withPermalink"><router-link :to="{ query: { id: `${data.map_type},${data.map_name},${data.hash_id}` } }" append>{{formatObjId(data.hash_id)}}</router-link></span>
       <span v-if="!withPermalink">{{formatObjId(data.hash_id)}}</span>
-      <span style="color: #ff3915" v-if="data.hard_mode"> Master Mode</span>
+    </section>
+    <section class="search-result-id">
+      <span style="color: #ff3915" v-if="data.hard_mode"><i class="fa fa-shield-alt fa-fw"></i> Master Mode</span>
+    </section>
+    <section class="search-result-id">
+      <span style="color: #ff3915" v-if="data.one_hit_mode"><i class="fa fa-shield-alt fa-fw"></i> One-hit Obliterator Object</span>
```
Comment (leoetlino) [inline src/components/ObjectInfo.vue:17]:
```suggestion
      <span style="color: #ff3915" v-if="data.one_hit_mode"><i class="fa fa-shield-alt fa-fw"></i> One-Hit Obliterator Object</span>
```

Also can we use different icons for the Master Mode and OhO indicators? (fa-shield-alt is already used for enemy equipment)

**Comment by Kaedenn** [comment]
Changed the OhO indicator to fa-infinity and capitalized the H.

File: src/components/ObjectInfo.vue [hunk 2]
```diff
@@ -9,7 +9,12 @@
       <i class="fas fa-hashtag fa-fw"></i> ID
       <span v-if="withPermalink"><router-link :to="{ query: { id: `${data.map_type},${data.map_name},${data.hash_id}` } }" append>{{formatObjId(data.hash_id)}}</router-link></span>
       <span v-if="!withPermalink">{{formatObjId(data.hash_id)}}</span>
-      <span style="color: #ff3915" v-if="data.hard_mode"> Master Mode</span>
+    </section>
+    <section class="search-result-id">
```
Comment (leoetlino) [inline src/components/ObjectInfo.vue:13]:
```suggestion
    <section class="search-result-hard-mode">
```

File: src/components/ObjectInfo.vue [hunk 3]
```diff
@@ -9,7 +9,12 @@
...
+      <span style="color: #ff3915" v-if="data.hard_mode"><i class="fa fa-dungeon fa-fw"></i> Master Mode</span>
+    </section>
+    <section class="search-result-id">
```
Comment (leoetlino) [inline src/components/ObjectInfo.vue:16]:
```suggestion
    <section class="search-result-oho">
```

**Review by leoetlino — COMMENTED** [review]
sorry, missed these last time

**Review by leoetlino — APPROVED** [review]

**Pull request merged**